web/html/src/manager/shared/spa/spa-engine.js

```diff
@@ -3,21 +3,28 @@ import "senna/build/senna.css"
 import "./spa-engine.css"
 import SpaRenderer from "core/spa/spa-renderer";
 
-var appInstance = new App();
-appInstance.setLinkSelector("a.js-spa");
-appInstance.setFormSelector("");
-appInstance.addSurfaces('left-menu-data');
-appInstance.addSurfaces('ssm-box');
-appInstance.addSurfaces('page-body');
-appInstance.addRoutes([{
-  path: /.*/,
-  handler: function (route, a, b) {
-    const screen = new HtmlScreen();
-    screen.setCacheable(false);
-    return screen;
-  }
-}]);
+window.pageRenderers = window.pageRenderers || {};
```

Comment:
Just to make sure we don't depend on other scripts and their order.

We are adding the SPA INIT function in the namespace "pageRenderers.spa.init" so it can be executed from outside only if the SPA engine flag is enabled